docs/js/search.js

```diff
@@ -0,0 +1,73 @@
+(function() {
+    function displaySearchResults(results, store) {
+      var searchResults = document.getElementById('search-results');
+      var baseURL = "/codewind";
+  
+      if (results.length) { // Are there any results?
+        var appendString = '';
+  
+        for (var i = 0; i < results.length; i++) {  // Iterate over the results
+          var item = store[results[i].ref];
+          appendString += `
+          <div class="card">

```

Comment:
I'd prefer as much of this HTML to be in a Jekyll HTML element if possible, this would make it faster and easier to make modifications in future.